src/app/domain/backup/models/Media.ts

```diff
@@ -0,0 +1,21 @@
+/**
+ * A media on the device
+ * @member {string} name
+ * @member {number} path
+ * @member {string} type
+ * @member {number} creationDate
+ */
+export interface Media {
+  name: string
+  path: string
+  type: 'image' | 'video'
+  creationDate: number
```

Comment:
no updateDate?

Reply:
Not for the moment but it will come when I will improve file identification. I choose to add field only when needed.